uber/tests/models/test_badge_funcs.py

```diff
@@ -172,6 +172,26 @@ def test_self_assignment(self, session):
         change_badge(session, session.staff_five,  c.STAFF_BADGE, new_num=5)
 
 
+class TestNonShiftChecks:
+    @pytest.fixture(autouse=True)
+    def set_shift_badges_off(self, monkeypatch):
+        monkeypatch.setattr(c, 'SHIFT_CUSTOM_BADGES', False)
+
+    @pytest.fixture(autouse=True)
+    def teardown_range_check(request):
+        return True  # TODO: make this actually check ranges
```

Comment:
Ah, gotcha.  A better solution would probably be to modify ``teardown_range_check`` to only do the check if ``c.SHIFT_CUSTOM_BADGES`` was truthy, then you wouldn't need to override it here.